### PR TITLE
Tng 546 nav mobile

### PR DESCRIPTION
**Pull request by rrieger-espirit**
In the previous version there was no navigation for any mobile devices, so we added the handling of a mobile navigation and its styling in this context.

### CODE REVIEW

File: src/components/Navigation/style.css [hunk 1]
```diff
@@ -68,3 +68,45 @@
 .Navigation--Mobile {
   height: 101px;
 }
+.Mobile{
```
Comment (tgolbs) [inline src/components/Navigation/style.css:71]: Please use the `Navigation--` prefix, when defining new classes, so no classes in other components are affected.

File: src/components/Navigation/index.tsx [hunk 2]
```diff
@@ -20,19 +20,29 @@ class Navigation extends BaseComponent<NavigationProps> {
...
+
   renderItem(item: NavigationItem, currentDepth: number) {
     const isActive = this.isActiveItem ? this.isActiveItem(item) : false;
     return (
       <li class="Navigation--Item">
         <a
-          class={`Navigation--Link ${isActive ? "active" : ""}`}
+          data-testid={`item-${item.id}`}
+          class={`Navigation--Link Nav--Mobile ${
+            isActive ? "active" : "inactive"
```
[the remark this answers is not in the document]
Comment (tgolbs) [reply src/components/Navigation/index.tsx:32]: You can replace the selectors with `:not(.active)`

**Review by tgolbs — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 3]
```diff
@@ -1,7 +1,44 @@
 import { Component } from "vue-property-decorator";
 import Vue from "vue";
 import Documentation from "./documentation";
+import { Navigation } from "./components";
```
[the remark this answers is not in the document]
Comment (rrieger-espirit) [reply src/App.tsx]: cleaned up

File: src/components/Navigation/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (rrieger-espirit) [reply src/components/Navigation/index.tsx]: replaced selectors and deleted ".inactive"

File: src/components/Navigation/style.css (hunk 1 above)
Comment (rrieger-espirit) [reply src/components/Navigation/style.css]: Added Prefix to its Navigation classes

File: src/components/Navigation/__tests__/Navigation.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+          },
+        ],
+      },
+    });
+    const navigation = container.querySelector(
+      ".Navigation--Mobile [data-testid='item-1']",
+    );
+    console.log(navigation);
+    await fireEvent(navigation!, new Event("click"));
+    expect(spy).toHaveBeenCalled();
```
[the remark this answers is not in the document]
Comment (rrieger-espirit) [reply src/components/Navigation/__tests__/Navigation.spec.tsx:39]: done

**Review by tgolbs — APPROVED** [review]

**Comment by tgolbs** [comment]
@rrieger-espirit Please resolve merge conflicts.

**Pull request merged**